.github/workflows/dart.yml

```diff
@@ -16,7 +16,7 @@ env:
 
 jobs:
   job_001:
-    name: "OS: linux; SDK: dev; PKG: mono_repo; TASKS: `cd ../ && dart mono_repo/bin/mono_repo.dart generate --validate`"
+    name: "linux; Dart dev; PKG: mono_repo; `cd ../ && dart mono_repo/bin/mono_repo.dart generate --validate`"
```

Comment:
While you are in here it would be nice to put the stage name for github, you can't see it otherwise and that provides usually a nice at a glance summary of what is happening.

Reply:
stage just for github though – right? Since it's already shown on travis

Reply:
Yes just for github

Reply:
done